cla_backend/apps/diagnosis/graph.py

```diff
@@ -137,88 +137,98 @@ def _get_id_default_dict_for(attr, attr_name="attr.name", element="node"):
                 }
             )
 
-    def process_nodes(self, is_templated=settings.DIAGNOSES_USE_TEMPLATES):  # noqa: C901
+    @staticmethod
+    def _get_text(a_node, is_templated, translate=True):
+        if is_templated and a_node.text and "{%" in a_node.text:
+            tpl = get_template_from_string(u"{%% load i18n %%}%s" % a_node.text)
+            if translate:
+                return lazy(lambda: tpl.render(Context()), text_type)()
+            with translation.override("en"):
+                return tpl.render(Context())
+        return a_node.text
+
+    @staticmethod
+    def _get_markdown(_text, is_templated):
+        if is_templated:
+            return lazy(lambda: markdown.markdown(force_text(_text)), text_type)()
+        return _text
+
+    def _get_node_data_value_or_default(
+        self, _node, key, as_type=None, translate=True, is_templated=settings.DIAGNOSES_USE_TEMPLATES
```

Comment:
Since `is_templated` is set by and is more or less hard coded `settings.DIAGNOSES_USE_TEMPLATES`, could you not remove this as a parameter and directly pass it to the second parameter of `self._get_text` on line 162